### ISSUE TITLE
Multiple licenses for one Office installation

### ISSUE THREAD

**Issue by benfdc** [MicrosoftDocs/microsoft-365-docs#2267]
An employee has a family copy of office at home. Can one copy of Office have to subscriptions so that the employee can use it for work while the rest of the household can use it for student and personal?

Does it make a difference if each user has their own login on the computer?

This is important practical information that ought to be in the FAQ. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 805a04c5-96c4-ee47-7746-3105e0b7fc45
* Version Independent ID: 9efa7447-6731-6435-d4fc-8ba81344e85a
* Content: [Understand subscriptions and licenses in Microsoft 365 for business](https://docs.microsoft.com/en-us/microsoft-365/commerce/licenses/subscriptions-and-licenses?view=o365-worldwide)
* Content Source: [microsoft-365/commerce/licenses/subscriptions-and-licenses.md](https://github.com/MicrosoftDocs/microsoft-365-docs/blob/public/microsoft-365/commerce/licenses/subscriptions-and-licenses.md)
* Service: **o365-administration**
* GitHub Login: @cmcatee-MSFT
* Microsoft Alias: **cmcatee**

**Comment by cmcatee-MSFT** [comment]
@benfdc Yes, the single installation of Office can be used with mixed accounts (Family vs a work or school account). It also makes a difference if each user has their own Windows login. If that's the case, Office will automatically use that login to authenticate, and they will use the correct license.

If the computer is a shared computer with only one Windows login, then each user must log in to Office with their own account, and then _log out when they are finished_. Otherwise, anyone who uses that computer will see Office files and settings for the person who last signed in.

**Comment by yogkumgit** [comment]
@cmcatee-MSFT  Thank you very much for sharing this explanation. @benfdc hope this comment is helpful for you. If you see a documentation update is required, please feel free to open an issue for the same. We proceed here to close it. Thanks for taking out some time to open the issue. Appreciate and encourage you to do the same in future also.
